src/VisualStudio/IntegrationTest/TestUtilities/InProcess/TextViewWindow_InProc.cs

```diff
@@ -94,11 +94,18 @@ public void WaitForLightBulbSession()
         /// querying the editor
         /// </remarks>
         public bool IsCompletionActive()
-            => ExecuteOnActiveView(view =>
+        {
+            if (!HasActiveTextView())
+            {
+                return false;
+            }
+
+            return ExecuteOnActiveView(view =>
```

Comment:
so, I guess before when there is no active  view, the complation active could return true? is that it? otherwise, for test, won't it be same?

Reply:
Previously it would throw an exception, which caused a test to fail in a recent integration test run.